src/shipit_static_analysis/tests/test_stats.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
```

Comment:
Here you're basically testing the mock, I guess it has less value but still it's good to make sure the mock keeps working in case we modify it somehow.